src/ocm/components/clusterConfiguration/CpuArchitectureDropdown.tsx

```diff
@@ -35,16 +48,20 @@ const fieldId = getFieldId(INPUT_NAME, 'input');
 
 const getInvalidCombinationReason = (
   featureSupportLevels: FeatureSupportLevelData,
-  cpuArchitecture: SupportedCpuArchitecture,
+  cpuArchitecture: CpuArchitecture,
```

Comment:
```suggestion
  cpuArchitecture: SupportedCpuArchitecture,
```

and you will be able to remove all the castings architectureData[cpuArchitecture] **as CpuArchitectureItem**